src/sqlfluff/core/templaters/slicers/tracer.py

```diff
@@ -239,7 +252,7 @@ def update_inside_set_or_macro(
             # When the second format is used, set the field
             # 'inside_set_or_macro' to True. This info is used elsewhere,
```

Comment:
It also can set `inside_block` instead so this comment is no longer 100% accurate.

Reply:
Will update.